src/tools/msc.lua

```diff
@@ -198,6 +198,10 @@
 		local shared = msc.getsharedflags(cfg)
 		local cflags = config.mapFlags(cfg, msc.cflags)
 		local flags = table.join(shared, cflags, msc.getwarnings(cfg))
+		-- Handle nodefaultlib "On" for compiler
+		if cfg.nodefaultlib == "On" then
+			table.insert(flags, "/Zl")
+		end
```

Comment:
Why isn't this just part of shared flags?

Reply:
Good catch. Originally, I had more here, then slimmed it back down because it was over-engineered.